mountpoint-s3-fs/Cargo.toml

```diff
@@ -48,6 +48,7 @@ tracing = { version = "0.1.41", features = ["log"] }
 tracing-log = "0.2.0"
 tracing-subscriber = { version = "0.3.19", features = ["env-filter"] }
 signal-hook = "0.3.17"
+csv = "1.3.1"
```

Comment:
We should make the dependency optional and only enable it with the "manifest" feature. Same with `rusqlite`.